### DISCUSSION TITLE
Diffrence between this repo and shinchiro repo?

### DISCUSSION THREAD

**Discussion by Obegg** [zhongfly/mpv-build#75] [Q&A] [answered]
As far as I understand the only differences are that this repo builds more (which is great), 
and has two patches (one from [patch_pr](https://github.com/zhongfly/mpv-winbuild/tree/main/patch_pr) and one from [patch](https://github.com/zhongfly/mpv-winbuild/tree/main/patch)),
Am I missing something here? Because if it's "better" (because it builds more and has two more patches) then why isn't it on https://mpv.io/installation/ ?

**Comment by zhongfly** [comment, marked as answer]
These 2 patches are only used to temporarily store patches for bugs that mpv-winbuild-cmake hasn't fixing yet, and are mostly empty.

When there is a build error and mpv-winbuild-cmake doesn't fix it, I put a temporary patch in `patch` to fix the error. Generally, the error is due to a dependency library update, so the patch is mostly used to roll back the dependent version.
`patch_pr` is currently never used in releases, it is only used for my personal testing of some of mpv's prs. If mpv has prs that have not been merged into the mainline and are not yet supported by mpv-winbuild-cmake, a special patch is put in here for building those prs only.

As for why it's not on that website, it's because this repository is somehow just a supplement to mpv-winbuild-cmake, and I personally don't have the desire or ability to independently maintain a toolchain for compiling.
